src/lib/interactionManager/authenticationFlow.ts

```diff
@@ -28,6 +28,8 @@ export class AuthenticationFlow extends Flow {
   }
 
   public async consumeAuthenticationRequest(token: Authentication) {
-    this.state.description = token.description
+    if (!this.state.description) this.state.description = token.description
```

Comment:
interaction is valid if a response has the same description, this is the only rule for auth iirc. the line below will also allow repeated responses with the same description, but that can be changed easily